source/enumerate-databases.rst

```diff
@@ -81,6 +81,22 @@ For example, to list only databases whose names begin with "foo":
 
   > db.adminCommand({listDatabases: 1, filter: {name: /^foo/}});
 
+AuthorizedDatabases
```

Comment:
I think this can be "Authorized Databases". I don't believe the section title maps to the option name. For instance, we have "Filters" for the previous section even though the option is `filter` (singular).